packages/core/test/card/cardTests.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { assert, expect } from "chai";
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import { Card, Classes } from "../../src/index";
+
+describe("<Card>", () => {
+    const noClass = (className: string) => `Card does not have "${className}" class`;
```

Comment:
this is nice but not necessary. I'd be fine with the assert message being just the `className`: we only need it to figure out _which_ assert failed.

`assert.isTrue(wrapper.hasClass(Classes.CARD), Classes.CARD)` will log `"pt-card"` on failure, which is enough info to debug.

Reply:
I will remove all exception messages from tests